### PR TITLE
Blog app: add devise

### PR DESCRIPTION
**Pull request by ogagaoghene**
# Blog app: add devise

## Description
Add devise gem for authentication

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -6,11 +6,15 @@
...
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css">
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %> 
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
+    <%= render 'shared/navbar' %>
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
     <%= yield %>
```
Comment (OLIPLICHE) [inline app/views/layouts/application.html.erb:18]: - Please make available the page(login) where to check if the codes are working: Here is the file where the errors is been noticed. 
- 
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi TEAM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the dev branch) 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (ogagaoghene) [reply app/views/layouts/application.html.erb:18]: @OLIPLICHE I am not getting this error locally. Attached are images as proof or evidence
image

image

image

image

Maybe you need to run rspec::webpack:install 

So you can fix that in your local environment

Please check again

Comment (ogagaoghene) [reply app/views/layouts/application.html.erb:18]: @OLIPLICHE 
bundle exec rails webpacker:install

Kindly help run the following command to update webpack locally at your end

This is from your end bro.

Comment (ogagaoghene) [reply app/views/layouts/application.html.erb:18]: @OLIPLICHE 
Could you please run the following commands
rails webpacker:install
bundle exec rake assets:precompile

Comment (OLIPLICHE) [reply app/views/layouts/application.html.erb:18]: Hello, I set my environment before running the App. If you sure about what you're are saying, Kindly submit. The second reviewer will check it. Sorry for late answer, I am outside, using my phone to reply.

**Review by danmainah — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (danmainah) [reply app/views/layouts/application.html.erb:18]: @ogagaoghene Running the commands provided didn't solve the issue, I did use  npm install, am used ubuntu to run the app, my observation `this was caused by your use of bootstrap CDN which required node modules to be setup`

**Comment by ogagaoghene** [comment]
@danmainah Thank you so much bro. I am eternally grateful to you. I appreciate your professional review. Thank and May God bless you real good.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (ogagaoghene) [reply app/views/layouts/application.html.erb:18]: @danmainah Thank you for the hint. I have learnt something. I am glad you were able to figure it out. Thanks bro.

**Pull request merged**